processing/bloom_handler.go

```diff
@@ -13,78 +12,82 @@ import (
 
 	"github.com/DCSO/fever/types"
 	"github.com/DCSO/fever/util"
+	"github.com/buger/jsonparser"
 
 	"github.com/DCSO/bloom"
 	log "github.com/sirupsen/logrus"
 )
 
 var sigs = map[string]string{
-	"http-url":  "%s Possibly bad HTTP URL: ",
-	"http-host": "%s Possibly bad HTTP host: ",
-	"tls-sni":   "%s Possibly bad TLS SNI: ",
-	"dns-req":   "%s Possibly bad DNS lookup to ",
-	"dns-resp":  "%s Possibly bad DNS response for ",
+	"http-url":  "%s Possibly bad HTTP URL: %s",
+	"http-host": "%s Possibly bad HTTP host: %s",
+	"tls-sni":   "%s Possibly bad TLS SNI: %s",
+	"dns-req":   "%s Possibly bad DNS lookup to %s",
+	"dns-resp":  "%s Possibly bad DNS response for %s",
 }
 
 // MakeAlertEntryForHit returns an alert Entry as raised by an external
 // indicator match, e.g. a Bloom filter hit. The resulting alert will retain
 // the triggering event's metadata (e.g. 'dns' or 'http' objects) as well as
 // its timestamp.
 func MakeAlertEntryForHit(e types.Entry, eType string, alertPrefix string, ioc string) types.Entry {
-	var eve types.EveEvent
-	var newEve types.EveEvent
-	var err = json.Unmarshal([]byte(e.JSONLine), &eve)
+
+	var value string
+	if eType == "http-url" {
```

Comment:
Nothing wrong, but maybe a switch is easier on the eyes :)

```go
switch {
case eType == ...:
case strings.HasPrrefix...:
}
```

Reply:
True. Done.